engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -1172,6 +1182,13 @@ public void init() throws IOException
         System.setProperty("socksProxyPort", socks.substring(colon + 1));
       }
     }
+    // set highlight color: check env var first and then config
+    String overrideColor = System.getenv(APEX_HIGHLIGHT_COLOR_ENV_VAR_NAME);
```

Comment:
I'd prefer to do lazy initialization: help is not that frequently used.
```
private String getHighlightColor() {
  if (highlightColor == null) {
    highlightColor = System.getenv(APEX_HIGHLIGHT_COLOR_ENV_VAR_NAME);
    if (StringUtils.isBlank(highlightColor)) {
      highlightColor = conf.get(APEX_HIGHLIGHT_COLOR_PROPERTY_NAME, FORMAT_BOLD);
    }
    highlightColor = color.replace("\\e", "\033");
  }
  return highlightColor;
}
```
This way you may also add unit test that verifies getHighlightColor() behavior.